### DISCUSSION TITLE
Capstone Project - Error Input Predtest

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#385] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78478403/193493763-d609eb2c-8628-43ac-93a8-644b01e3dc0d.png)

Selamat Pagi kak permisi, saya izin bertanya mengenai capstone project, saya coba menggunakan random forest dalam kasus Concrete-Prediction Will it last forever?, nah saya coba mengimplementasikannya kedalam data validasi, kebetulan errornya cukup kecil, ketika saya mencoba memasukkan kedalam data-test itu muncul error missing values in newdata, itu kenapa ya kira-kira? ketika saya mencoba menggunakan linear regresi tidak muncul error
*data-testnya memang file kosong saya ingin menginputkan modelnya kedalam situ, namun ketika menggunakan model linear regresi baik baik saja 

berikut untuk kodenya 
library(randomForest)
set.seed(100)
model_rf <- randomForest(x = data_train %>% select(-strength),
                         y = data_train$strength,
                         ntree = 500)

model_rf

**Comment by VictorNugraha** [comment, marked as answer]
Halo @daryraa,

Boleh diperlihatkan print out dari data testnya?

Best Regards,

Team Algoritma

**Reply by daryraa** [reply]
maaf print out bagaimana mas?

**Reply by daryraa** [reply]
![image](https://user-images.githubusercontent.com/78478403/194578164-1232ede2-78a3-4dba-b063-c5ed644efb89.png)
ini bukan mas?

**Reply by daryraa** [reply]
namun ketika menggunakan model ini model_rf <- randomForest(formula = strength ~., data = data_train)
bisa dimasukkan kedalam data-test

**Reply by jafarhf1** [reply]
Halo @daryraa, terkait kendala yang bapak alami, bapak jangan mengikutsertakan kolom "strength" saat melakukan prediksi karena kolom tersebut adalah variabel targetnya.

Demikian yang bisa kami sampaikan. Apabila jawaban ini membantu, silahkan mark as answer.
